.github/workflows/post_release_version_bump.yml

```diff
@@ -12,21 +12,12 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
-      - name: Install Python
-        uses: actions/setup-python@v2
+      - name: Run post-release version bump
+        uses: PennyLaneAI/automation/version_bump_action@v1.0
```

Comment:
Wondering if as per our chat, we should pin to `master` here instead of `v1.0`. Also found this (though may not be relevant anymore):
https://github.com/actions/upload-artifact/issues/41